fastlane/lanes/release.rb

```diff
@@ -148,42 +151,44 @@
   #
   desc 'Trigger a new beta build on CI'
   lane :new_beta_release do |options|
+    ensure_git_branch_is_release_branch
+
     # Verify that there's nothing in progress in the working copy
     ensure_git_status_clean
 
-    # Verify that the current branch is a release branch. Notice that `ensure_git_branch` expects a RegEx parameter
-    ensure_git_branch(branch: '^release/')
-
     git_pull
 
-    # Check versions
-    message = <<-MESSAGE
-      • Current build code: #{build_code_current}
-      • New build code: #{build_code_next}
+    skip_user_confirmation = options[:skip_confirm]
 
-      • Current internal build code: #{build_code_current_internal}
-      • New internal build code: #{build_code_next_internal}
-    MESSAGE
+    unless skip_user_confirmation
+      # The `release_version_next` is used as the `new internal release version` value because the external and internal
+      # release versions are always the same.
+      message = <<-MESSAGE
+        • Current build code: #{build_code_current}
+        • New build code: #{build_code_next}
 
-    UI.important(message)
-    UI.user_error!('Aborted by user request') unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
+        • Current internal build code: #{build_code_current_internal}
+        • New internal build code: #{build_code_next_internal}
+      MESSAGE
+
+      UI.important(message)
```

Comment:
Similar to a previous comment, how do you feel about always printing the `message` outside of the `skip_user_confirmation` check to make the information available in CI logs?